### ISSUE TITLE
Permission denied after fresh installation

### ISSUE THREAD

**Issue by Signum** [rbenv/rbenv#1472]
I have cloned the repository on a Linux Mint 20.3 system as described in the installation instructions:

    git clone https://github.com/rbenv/rbenv.git ~/.rbenv

Then I have added the necessary lines to my ~/.zshrc:

    echo 'eval "$(~/.rbenv/bin/rbenv init - zsh)"' >> ~/.zshrc

I opened up a new shell and was greeted by:

    /home/signum/.zshrc:138: permission denied: /home/signum/.rbenv/bin/rbenv

The ~/.rbenv/bin/rbenv file is a text file consisting of:

    ../libexec/rbenv

**Comment by mislav** [comment]
Hi, the `bin/rbenv` file in our repository is a symlink to `../libexec/rbenv`. It seems that, when you checked out this repository with git on your system, git created a plain file instead of a real symlink on your system.

1. Does your filesystem support symlinks?
2. What is the setting `git config core.symlinks`?

**Comment by Signum** [comment]
My core.symlinks was set to "false". I have no idea why (or if) I did that. After changing that back to the default (true) rbenv worked fine again. Very good guess and thanks for the swift help.

**Comment by stefanos82** [comment]
> My core.symlinks was set to "false". I have no idea why (or if) I did that. After changing that back to the default (true) rbenv worked fine again. Very good guess and thanks for the swift help.

When I changed my `core.symlinks` to `true`, it affected all my repos I was working on; it wanted me to push changes on files I never worked on them and it took me a while to realize what actually happened.

As soon as I changed symlinks back to `false`, everything worked as expected.

So, to fix this `rbenv` issue, all I had to do was to remove the `rbenv` file and run `ln -s ../libexec/rbenv` to re-create it without messing git's global settings.

In other words, tldr;

- `cd $HOME/.rbenv/bin/`
- `rm rbenv`
- `ln -s ../libexec/rbenv`

That should be it.
